src/docs/platform-automation/alertmanager.md

```diff
@@ -76,29 +96,36 @@ Alerts are triggered only if the condition persists for 1 hour, minimizing unnec
 
 In the **Developer** perspective, select **Observe** -> **<project_name>** -> **Alerts**. In this perspective, alerts, silences, and alerting rules are all managed from the **Alerts** page. The results shown in the **Alerts** page are specific to the selected project.
 
-## Objects
+## Silencing Alerts
```

Comment:
```suggestion
## Silencing alerts
```

Pilar says best practice is to capitalize only the first word in a title (or proper nouns, etc)